docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
 
+Moreover, you can use filters that allow you to quickly filter your workload as per labels, field selectors, or [CEL expression](https://kubernetes.io/docs/reference/using-api/cel/) as shown below.
 
-### Create Kubernetes Resource
+{% embed url="https://www.youtube.com/watch?v=E-V-ELCXtfs" caption="Filtering Workloads in Devtron" %}
 
-**Note**: As a pre-requisite, you must have a basic understanding of Kubernetes Cluster, Resources, Kinds.
+### Edit Manifest 
 
-You can create a Kubernetes resource by passing definition YAML file. You can create more than one resource by separating the resource YAMLs by ‘---’.
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to edit a manifest.
+{% endhint %}
 
-An example that shows the required fields and object specifications for a Kubernetes Deployment:
+You can edit the [manifest](../reference/glossary.md#manifest) of a Kubernetes object. This can be for fixing errors, scaling resources, or changing configuration.
+
+![Figure 5: Editing a Live Manifest](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/edit-live-manifest.gif)
+
+### View Events
+
+You can monitor activities like creation, deletion, updation, scaling, or errors in the resources involved.
+
+![Figure 6: Viewing All Events](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/events.gif)
+
+### Delete
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to delete a resource.
+{% endhint %}
+
+You can delete an unwanted resource if it is orphaned and no longer required by your applications.
+
+![Figure 7: Deleting a Resource](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/delete.gif)
+
+You can perform all the actions on the following:
+* [Nodes](#nodes)
+* [Pods](#pods)
+* [Other Resource Kinds](#other-resource-kinds)
+
+---
+
+## Nodes
+
+You can see the list of nodes available in your cluster. Typically you have several nodes in a cluster; in a learning or resource-limited environment, you might have only one node.
+
+The components on a typical node include the `kubelet`, a `container runtime`, and the `kube-proxy`.
+
+If you have multiple nodes, you can search a node by name or label in the search bar. The search result will display the following information of the node. To display a parameter of a node, use `Columns` on the right side, select the parameter to display from the drop-down list, and click **Apply**.
+
+![Figure 8: Searching and Filtering Nodes](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/cluster-nodes.jpg)
+
+| Fields | Description |
+| --- | --- |
+| Node | User-defined name for the node in Devtron. E.g. `demo-new`.<br>Note: Two nodes cannot have the same name at the same time.</br> |
```

Comment:
How can a user define name of a node on devtron?

Reply:
Fixing it. Got leaked from our existing doc, I suppose (snapshot attached below)

<img width="1426" alt="image" src="https://github.com/devtron-labs/devtron/assets/141001279/52a9ca9f-097c-40a0-82e5-e615b909c323">